tests/Feature/Modules/Core/BillCrudTest.php

```
@@ -0,0 +1,220 @@
+
+    protected Currency $currency;
+
+    protected Partner $partner;
+
+    protected Journal $journal;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        Model::unguard();
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Database\Eloquent\Model' in method 'setUp'. (undefined)

(StaticAccess)